stackbrew-config.yaml

```diff
@@ -1,14 +1,14 @@
 versions:
-  - caddy_version: '2.8.0-rc.1'
-    is_major: false
-    is_latest: false
-    dist_commit: 509c30cecd3cbc4012f6b1cc88d8f3f000fb06e4
-  - caddy_version: '2.7.6'
+  - caddy_version: '2.8.0'
     is_major: true
     is_latest: true
     dist_commit: 509c30cecd3cbc4012f6b1cc88d8f3f000fb06e4
+  - caddy_version: '2.7.6'
```

Comment:
Might even be worth dropping 2.7.6 entirely here

Reply:
Done